build.bash

```diff
@@ -2,7 +2,7 @@
 set -e
 
 ####################################################################
-#### dummy: changed this line 18 times to force another image build
+#### dummy: changed this line 19 times to force another image build
```

Comment:
BTW we often don't need this. There's a 'build' action in Github.

Reply:
I know, I actually only changed that so I could issue a new commit and rebase it to master.